bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -157,8 +157,13 @@ object BagsState {
 
     // Successful AGS lookup while running (and not edited).  Apply the update
     // and move to IdleState.
-    private[ags] override def succeed(results: Option[AgsStrategy.Selection]): StateTransition =
-      (IdleState(k, Some(hash)), BagsManager.applyAction(k, results))
+    private[ags] override def succeed(results: Option[AgsStrategy.Selection]): StateTransition = {
+      val h = results.fold(hash) { r =>
+        if (r.posAngle === ctx.getPositionAngle) hash
```

Comment:
I'm fine with promoting it if it seems useful.  My hesitation is that it always depends on context when two things are considered close enough.  Not even `Angle` has an `AlmostEqual` instance that works in every use case.  Maybe nothing does really?

At any rate, it's a one-liner  (`(a.toDegrees - b.toDegrees).abs < 0.0001` or something) so we could just type it out here.